src/main/scala/com/newlibertie/pollster/impl/CryptographicParameters.scala

```diff
@@ -23,6 +23,12 @@ case class CryptographicParameters
 )
 {
   val public_key_h = generator_g.modPow(private_key_s, large_prime_p)
+
+  override def toString: String = {
+    s"""
```

Comment:
same this is not a general to string (see similar comments below)

Reply:
overwrite to string only within spirit of its original meaning : "to display data to humans"